cypress/e2e/feeder-rss-import.cy.ts

```diff
@@ -0,0 +1,47 @@
+describe("Feeder RSS Import", () => {
+  before(() => {
```

Comment:
This should be `beforeEach`, otherwise additional tests will fail (ask me how I know).